tests/unit/lms/services/jstor_test.py

```diff
@@ -63,56 +63,108 @@ def test_via_url_with_bad_return_value_from_s3(
             svc.via_url(pyramid_request, document_url="jstor://ANY")
 
     @pytest.mark.parametrize(
-        "article_id, api_response, expected_api_url, expected_metadata",
```

Comment:
This test has been split into two: One test that checks we call the JSTOR API with the expected parameters, and a separate one that checks the response is parsed as expected.

Reply:
:+1: Yep, I think this is a good move.

Reply:
I can't make a suggestion here, but I think we can tighten up this layout by removing the trailing commas and re-running `make format`.

`black` will respect your trailing commas if you put them in, so we end up with overly spaced out code. This would have been required before, but now all these fixtures and the method args back-up onto one line nicely.